detox/ios/Detox/DetoxInstrumentsManager.m

```diff
@@ -147,6 +147,7 @@ - (id)_configForDetoxRecordingWithURL:(NSURL*)URL
 	
 	//TODO: Finalize the actual config for Detox perf recording.
 	[config setRecordNetwork:YES];
+	[config setRecordInternalReactNativeEvents:YES];
```

Comment:
This will not compile. You need to expose the property.

Reply:
Also, it's probably best to wrap this with `respondsToSelector:`, because this is a very recent feature and maybe One App has an old profiler.